### ISSUE TITLE
Input allow only first filetype

### ISSUE THREAD

**Issue by Micha-he** [tuyafeng/Via#422]
If I open a upload-file-request at the homepage of my health insurance, JPG, JPEG and TIFF files are not allowed, but PDF is! Other browsers all work as required, with all allowed filetypes.

I mean, this is the relevant position on their website:

&lt;input id="f1097058d1098062" name="auUpload" class="" value="" placeholder='' type="file" accept="application/pdf,image/jpeg,image/jpg,image/tiff" title="Arbeitsunfähigkeitsbescheinigung"&gt;

Website: www.knappschaft.de
My phone: Nokia 6.1, Android One 9

**Comment by LakorTi** [comment]
Interesting bug, i wll fix it if i can.

**Comment by tuyafeng** [comment]
In the latest version(3.9.10), I have fixed this bug, you can see it after the update. If you still have problems, please feel free to reopen this issue.

**Comment by Micha-he** [comment]
I can't reopen this issue, but the error exist also in the new version 3.9.10 !

**Comment by tuyafeng** [comment]
Unfortunately, I forgot to solve the problem in another situation, and I will fix it later:joy::joy:

**Comment by Micha-he** [comment]
Same on this website:

https://brouter.de/brouter-web

with menue 'load/tours'. I can't select any filetype.

**Comment by tuyafeng** [comment]
In the latest version(4.0.2), I have fixed this bug, you can see it after the update. I have tested it on different websites and there should be no problems.

**Comment by Micha-he** [comment]
No, not working here with V4.0.2 and e.g. with the brouter-site.

**Comment by Micha-he** [comment]
New test with V4.0.7:
On website 'www.knappschaft.de' the jpg-filetype works. On the other two example-websites, the gpx-files from my internal storage are not available for upload :-(
